pep-0696.rst

```diff
@@ -0,0 +1,396 @@
+PEP: 696
+Title: Type defaults for TypeVars
+Version: $Revision$
+Last-Modified: $Date$
+Author: James Hilton-Balfe <gobot1234yt@gmail.com>
+Sponsor: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Discussions-To: typing-sig@python.org
+Status: Draft
+Type: Standards Track
+Topic: Typing
+Content-Type: text/x-rst
+Created: 14-Jul-2022
+Python-Version: 3.12
+
+Type defaults for TypeVars
+==========================
+
+Abstract
+--------
+
+This PEP introduces the concept of type defaults for TypeVars, which act
+as defaults for a type parameter when none is specified.
+
+Motivation
```

Comment:
In PEP 695, I included a survey of generics across multiple programming languages. I intentionally included mention of whether each language supports default type arguments to help motivate this PEP. If you think it would be helpful, you could refer to Appendix A in PEP 695 and mention that default type arguments are provided by several other languages.